appknox/analyses.go

```diff
@@ -117,7 +119,12 @@ func (s *AnalysesService) ListByFile(ctx context.Context, fileID int, opt *Analy
 	var drfResponse DRFResponseAnalysis
 	_, err = s.client.Do(ctx, req, &drfResponse)
 	if err != nil {
-		return nil, nil, err
+		if strings.Contains(err.Error(), "404") {
+			fileId := strconv.Itoa(fileID)
+			return nil, nil, errors.New("Analyses for fileID " + fileId + " doesn’t exist. Are you sure " + fileId + " is a fileID?")
+		} else {
```

Comment:
This else is not required just return it

Reply:
Ahh! my bad.. Removed else 👍🏼